projects/workflows-creator/src/lib/builder/group/group.component.ts

```diff
@@ -590,6 +590,17 @@ export class GroupComponent<E> implements OnInit, AfterViewInit {
     }
   }
 
+  handleKeyPress(event:any) {
```

Comment:
A small suggestion :
```javascript
handleKeyPress(event: any) {
  const keyCode = event.which || event.keyCode;
  const isDigit = keyCode >= 48 && keyCode <= 57;
  const isBackspaceOrDelete = [8, 46].includes(keyCode);
  const inputValue = event.target.value;
  const isValidInput = /^-?\d*\.?\d*$/.test(inputValue);

  if (!(isDigit || isBackspaceOrDelete) || !isValidInput) {
    event.preventDefault();
  }
}
```
we can combine the two event.preventDefault() calls into a single conditional statement.

Reply:
sure sir.

Reply:
done ?? @Deepika516

Reply:
I  had a talk with @rohit-sourcefuse sir @yeshamavani he said no need to do this right now , its a suggestion.

Reply:
Done